File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.java
import java.util.*;

class Solution {
    public String solution(String s) {
        
        StringBuilder sb = new StringBuilder();
        boolean start = true;
        
        for(int i=0; i<s.length(); i++){
            if(s.charAt(i) == ' '){
                start = true;
                sb.append(" ");
            }else{
                if(start){
                    sb.append(Character.toUpperCase(s.charAt(i)));
                    start = false;
                }else{
                    sb.append(Character.toLowerCase(s.charAt(i)));
                }
            }
        }
        
        return sb.toString();
        
    }
}

File: 프로그래머스/2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.js
function solution(s) {
    
    var answer = '';
    var first = true;
    
    for(let i=0; i<s.length; i++){
        if(s.charAt(i) == " "){
            first = true;
            answer += " ";
        }else{
            if(first){
                answer += s.charAt(i).toUpperCase();
                first = false;
            }else{
                answer += s.charAt(i).toLowerCase();
            }
        }
    }
    
    return answer;
}